cmake/graph-compiler.cmake

```diff
@@ -32,7 +33,14 @@ if (NOT DEFINED GRAPH_COMPILER_LIBS)
             GcJitWrapper
             GcCpuRuntime
     )
+
+    if (GC_ENABLE_IMEX)
+        add_definitions(-DGC_ENABLE_IMEX)
```

Comment:
It should be -DGC_USE_IMEX, as it's defined in the GC code and you don't need to add it explicitly, it's already defined in GRAPH_COMPILER_COMPILE_OPTIONS below.